tests/test_formatter.py

```diff
@@ -107,8 +107,13 @@ def test_timestamp(self):
 
         timestamp = time.time()
         dt = datetime.utcfromtimestamp(timestamp)
+
         assert self.formatter._format_token(dt, "X") == str(int(timestamp))
 
+        # time.time() may return a float with greater than 6 digits of precision
+        rounded_ts = str(round(timestamp * 1000000))
+        assert self.formatter._format_token(dt, "x") == rounded_ts.format("{f}")
```

Comment:
Any reason you chose to do `self.formatter._format_token(dt, "x") == rounded_ts.format("{f}")` over just `self.formatter._format_token(dt, "x") == rounded_ts`?

Reply:
Without that format it defaults to an exponential style which causes an AssertionError.

```
E       AssertionError: assert '1585764864000000' == '1.5857648648e+15'
E         - 1585764864000000
E         + 1.5857648648e+15
```

Reply:
Oh interesting. Did not realize that :D.